### ISSUE TITLE
val_loss begin increasing from the beginning of training

### ISSUE THREAD

**Issue by haoqi** [fchollet/keras#1563]
I build a basic multi-task NN by Graph(). On the top layer, there are 5 softmax output layers, referring 5 classification problems, the training labels are encoded as 1 of k(k =3 in each task). 
However, when training the system, the val_loss will increase from the beginning,  any idea?
I was think if it is the unbalanced training data issue, since for each 3 classed, the data percentage is 25%, 25%,50%, but not sure, because even if i change the validation_split to a larger number, it still happens. The loss is using 'categorical_crossentropy'

Using Theano backend.
105.0
Training...
Train on 61148 samples, validate on 15288 samples
Epoch 1/50
61148/61148 [==============================] - 1s - loss: 4.7502 - val_loss: 5.5295
Epoch 2/50
61148/61148 [==============================] - 1s - loss: 4.1619 - val_loss: 5.8611
Epoch 3/50
61148/61148 [==============================] - 1s - loss: 3.8825 - val_loss: 6.2142
Epoch 4/50
61148/61148 [==============================] - 1s - loss: 3.6600 - val_loss: 6.3189
Epoch 5/50
61148/61148 [==============================] - 1s - loss: 3.4832 - val_loss: 6.5152

**Comment by ymcui** [comment]
Hi,
i think you'd better set `show_accuracy=True`, so that you will figure out what is going on in your validation set, because what you are expecting is the `accuracy`, not `loss`.
Also, as far as i know, sometimes the `loss` goes up, but the `acc` will also goes up.

**Comment by haoqi** [comment]
I supposed to do that, however, Graph() does NOT support  show_accuracy=True.

**Comment by ymcui** [comment]
Yep, so you'd better write a function for calculating accuracy. :)

**Comment by keunwoochoi** [comment]
Unbalanced set itself is usually okay only if you also have the equally unbalanced train/valid/test set. What happens after then? 1:1:2 seems not that bad, val_loss should decrease learning goes correctly I think

**Comment by haoqi** [comment]
Finally, the problem might come from unshuffled validation dataset, since keras, just use last certain portion of the training data as validation set, if you specify the validation_split. So the validation set will not get from the shuffled dataset. Finally, I choose the validation set by myself. Thanks guys!!

**Comment by ersinyar** [comment]
@haoqi Did you manage to obtain decreasing values for val_loss over epochs? If yes, is it by specifying the validation set yourself or with another way?
